tests/test_holiday_base.py

```diff
@@ -1189,3 +1189,87 @@ def test_get_working_days_count(self):
         self.assertEqual(self.hb.get_working_days_count("2024-04-29", "2024-05-04"), 3)
         self.assertEqual(self.hb.get_working_days_count("2024-04-29", "2024-05-05"), 3)
         self.assertEqual(self.hb.get_working_days_count("2024-04-29", "2024-05-06"), 4)
+
+
+class TestNextHoliday(unittest.TestCase):
+    def setUp(self):
+        self.thisYear = datetime.now().year
```

Comment:
Please use snake-case

Reply:
Done - how about adding this to your ruff config?